index.js

```diff
@@ -42,6 +47,12 @@ module.exports = {
     // Place spaces after keywords
     "keyword-spacing": "error",
 
+     // Allow mixing operators. Risky, but required for prettier.
+    "no-mixed-operators": "off",
```

Comment:
An alternative would be to use a `prettier-ignore` line in these places.  Not sure what the violations look like specifically, I'll leave it up to your judgement whether or not that makes sense.